pkg/mcs/resource_manager/server/resource_group.go

```diff
@@ -256,18 +192,11 @@ func (rg *ResourceGroup) SetStatesIntoResourceGroup(states *GroupStates) {
 	switch rg.Mode {
 	case rmpb.GroupMode_RUMode:
 		if state := states.RU; state != nil {
-			rg.RUSettings.RU.GroupTokenBucketState = *state
+			rg.RUSettings.RU.setState(states.RU)
```

Comment:
```suggestion
			rg.RUSettings.RU.setState(state)
```

Reply:
for RU maybe just need to set `states.RU`